### DISCUSSION TITLE
blog/pictures-of-canada/

### DISCUSSION THREAD

**Discussion by giscus** [svandegar/seraphin#6] [Announcements]
# blog/pictures-of-canada/

The scenic lands of Canada featuring maple leaves, snow-capped mountains, turquoise lakes and Toronto. Take in the sights in this photo gallery exhibition and see how easy it is to replicate with some MDX magic and tailwind classes.

http://localhost:3000/blog/pictures-of-canada/

**Comment by svandegar** [comment]
Coucou
